src/main/java/org/broadinstitute/hellbender/engine/filters/FragmentLengthReadFilter.java

```diff
@@ -8,15 +8,17 @@
 import java.io.Serializable;
 
 /**
- * Keep reads that are within a given max fragment length.
+ * Keep only read pairs with fragment length at most equal to the given value.
+ *
+ * <p>Note: Fragment length refers to the insert size (absolute difference between the 5'-end of the read and the mate).
```

Comment:
Done. By the way, it will be interesting to include a summary in the `@DocumentedFeature` annotation, because now descriptions are getting really long. I will add a separate commit after addressing your comments.